toppra/dracula/three_segment_spline.py

```diff
@@ -0,0 +1,154 @@
+from __future__ import print_function
+
+from numpy.testing import assert_almost_equal
+
+import copy
+import inspect
+import math
+import os
+
+import numpy as np
+import matplotlib.pyplot as plt
+import argparse
+import logging
+logging.basicConfig()
+
+# import time
+# from scipy.interpolate import CubicSpline
+from scipy.interpolate import PPoly
+
+# %matplotlib inline
+
+#### Calculations for stretching one segment:
+def ThreeSegmentSpline(Ps, Vs, As, Pe, Ve, Ae, Jmax, fin = None, SafetyTime = 1.70):
```

Comment:
Are these standard argument names inside toppra? They're pretty opaque from a user's perspective. Could you add some documentation?

Reply:
commented